tests/integration/targets/dcnm_vrf/tests/dcnm/self-contained-tests/vrf_lite.yaml

```diff
@@ -0,0 +1,286 @@
+##############################################
+##                 SETUP                    ##
+##############################################
+
+- set_fact:
+    rest_path: "/rest/control/fabrics/{{ test_fabric }}"
+  when: controller_version == "11"
+
+- set_fact:
+    rest_path: "/appcenter/cisco/ndfc/api/v1/lan-fabric/rest/control/fabrics/{{ test_fabric }}"
+  when: controller_version >= "12"
+
+- name: MERGED - Verify if fabric is deployed.
+  cisco.dcnm.dcnm_rest:
+    method: GET
+    path: "{{ rest_path }}"
+  register: result
+
+- assert:
+    that:
+    - 'result.response.DATA != None'
+
+- name: MERGED - Clean up any existing vrfs
+  cisco.dcnm.dcnm_vrf:
+    fabric: "{{ test_fabric }}"
+    state: deleted
+
+- name: VRF LITE - sleep for 40 seconds for DCNM to completely update the state
+  wait_for:
+    timeout: 40
+
+###############################################
+###                TESTS                     ##
+###############################################
+
+- name: VRF LITE- Create, Attach and Deploy new VRF - VLAN/VRF LITE EXTENSION Provided by the User in one switch
+  cisco.dcnm.dcnm_vrf: &conf1
+    fabric: "{{ test_fabric }}"
+    state: merged
+    config:
+    - vrf_name: ansible-vrf-int1
+      vrf_id: 9008011
+      vrf_template: Default_VRF_Universal
+      vrf_extension_template: Default_VRF_Extension_Universal
+      vlan_id: 500
+      source: null
+      service_vrf_template: null
+      attach:
+      - ip_address: "{{ ansible_switch1 }}"
+      - ip_address: "{{ ansible_switch2 }}"
+        vrf_lite:
+          - peer_vrf: ansible-vrf-int1 # optional
+            interface: "{{ ansible_int1 }}" # optional
+            ipv4_addr: 10.33.0.2/30 # optional
+            neighbor_ipv4: 10.33.0.1 # optional
+            ipv6_addr: 2010::10:34:0:7/64 # optional
+            neighbor_ipv6: 2010::10:34:0:3 # optional
+            dot1q: 2 # dot1q can be got from dcnm
+      deploy: true
+  register: result
+
+- name: Query fabric state until vrfStatus transitions to DEPLOYED state
+  cisco.dcnm.dcnm_vrf:
+    fabric: "{{ test_fabric }}"
+    state: query
+  register: query_result
+  until:
+    - "query_result.response[0].parent.vrfStatus is search('DEPLOYED')"
+  retries: 30
+  delay: 2
+
+- assert:
```

Comment:
Are we checking for vrf-lite specifically in this assert statement?  Based on the task above I would expect vrf-lite to only be associated with `switch2` and not `switch1`.  Is that correct?  We should try and explicitly check for vrf-lite config/behavior in the assert here and in other tests in the file

Reply:
Changes done to check for vrf-lite configurations for the associated switch.